docs/user_guide/faq.rst

```diff
@@ -43,14 +50,15 @@ How do I...
 ... change the logger level?
     .. code-block:: python
 
-        chip = siliconcompiler.Chip('<design>', loglevel=<INFO|ERROR|DEBUG>)
+        chip = siliconcompiler.Chip('<design>', loglevel=<INFO|WARNING|ERROR|DEBUG>)
```

Comment:
nitpick: `INFO|DEBUG|WARNING|ERROR`?

Reply:
I can go either way, any reason for that order?

Reply:
I think it would be in order of lowest->highest priority, so setting any value would hide every message type to its left.